contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelItem.vue

```diff
@@ -374,7 +384,11 @@
       copyToken: 'Copy channel token',
       deleteChannel: 'Delete channel',
       deleteTitle: 'Delete this channel',
+      removeChannel: 'Remove from channel list',
+      removeTitle: 'Remove from channel list',
       deletePrompt: 'This channel will be permanently deleted. This cannot be undone.',
+      removePrompt:
+        "You have view-only access to this channel. Removing it will hide it from your list and will not affect anyone else's access",
```

Comment:
Pleas update this string to what was suggested [here](https://github.com/learningequality/studio/pull/4918#issuecomment-2807590020) by Radina